presidio-analyzer/tests/test_credit_card_recognizer.py

```diff
@@ -1,184 +1,148 @@
-from analyzer import matcher, common_pb2
-from tests import *
-import os
-
 # https://www.datatrans.ch/showcase/test-cc-numbers
 # https://www.freeformatter.com/credit-card-number-generator-validator.html
+from analyzer.predefined_recognizers import CreditCardRecognizer
 
-fieldType = common_pb2.FieldTypes()
-fieldType.name = common_pb2.FieldTypesEnum.Name(common_pb2.CREDIT_CARD)
-types = [fieldType]
+credit_card_recognizer = CreditCardRecognizer()
+entities = ["CREDIT_CARD"]
 
 
 def test_valid_credit_cards():
+    # init
     number1 = '4012888888881881'
     number2 = '4012-8888-8888-1881'
     number3 = '4012 8888 8888 1881'
-    results = match.analyze_text('{} {} {}'.format(number1, number2, number3),
-                                 types)
-
-    assert len(results) == 3
-    assert results[0].text == number1
-    assert results[0].score == 1.0
-    assert results[1].text == number2
-    assert results[1].score == 1.0
-    assert results[2].text == number3
-    assert results[2].score == 1.0
-
 
-def test_valid_credit_cards_with_lemmatized_context():
-    number1 = '4012888888881881'
-    number2 = '4012-8888-8888-1881'
-    number3 = '4012 8888 8888 1881'
-    context = 'my credit cards are:'
-    results = match.analyze_text(
-        '{}{} {} {}'.format(context, number1, number2, number3), types)
+    results = credit_card_recognizer.analyze_all('{} {} {}'.format(number1, number2, number3), entities)
 
     assert len(results) == 3
-    assert results[0].text == number1
+    assert results[0].entity_type == "CREDIT_CARD"
     assert results[0].score == 1.0
-    assert results[1].text == number2
     assert results[1].score == 1.0
-    assert results[2].text == number3
     assert results[2].score == 1.0
 
 
 def test_valid_airplus_credit_card():
     number = '122000000000003'
-    results = match.analyze_text(number, types)
+    results = credit_card_recognizer.analyze_all(number, entities)
```

Comment:
regarding my prev comment, i think analyze_all can be changed to analyze, it will look better.

also, when i think about it, what happens if instead of analyze all I call analyze? its a bit confusing